pybamm/models/submodels/thermal/lumped.py

```diff
@@ -67,4 +67,4 @@ def set_rhs(self, variables):
 
     def set_initial_conditions(self, variables):
         T_vol_av = variables["Volume-averaged cell temperature [K]"]
-        self.initial_conditions = {T_vol_av: self.param.T_init}
+        self.initial_conditions = {T_vol_av: pybamm.x_average(self.param.T_init)}
```

Comment:
will also need to do this in the "x-lumped" models for pouch cell (in the folder `submodels/thermal/pouch_cell/`)